lizmap/modules/lizmap/classes/qgisExpressionUtils.class.php

```diff
@@ -156,10 +156,13 @@ public static function updateExpressionByUser($layer, $expression, $edition = fa
      * @param string          $filter        A filter to restrict virtual fields creation
      * @param string          $withGeometry  'true' to get geometries of features
      * @param string          $fields        A list of field names separated by comma. E.g. 'name,code'
+     * @param int             $limit         The maximum number of features to return
+     * @param null|string     $sortingField  The field to sort by E.g. 'desc'
+     * @param null|string     $sortingOrder  The order to sort by E.g. 'name'
      *
      * @return null|array the features with virtual fields
      */
-    public static function virtualFields($layer, $virtualFields, $filter = null, $withGeometry = 'true', $fields = '')
+    public static function virtualFields($layer, $virtualFields, $filter = null, $withGeometry = 'true', $fields = '', $limit = 1000, $sortingField = null, $sortingOrder = null)
```

Comment:
```suggestion
    public static function virtualFields($layer, $virtualFields, $filter = null, $withGeometry = 'true', $fields = '', $limit = 1000, $sortingField = null, $sortingOrder = 'asc')
```
As `sorting_order` parameter default value